openshift/templates/rails-postgresql-persistent.json

```diff
@@ -23,11 +23,7 @@
       "kind": "Secret",
       "apiVersion": "v1",
       "metadata": {
-        "name": "${NAME}",
-        "annotations": {
-          "template.openshift.io/expose-username": "{.data['application-user']}",
-          "template.openshift.io/expose-password": "{.data['application-password']}"
-        }
```

Comment:
@jim-minter ptal, we don't have the db creds exposed in our other example templates so i'm inclined to remove it here for consistency...are you aware of any reason why this one had it but the others do not?

e.g. https://github.com/openshift/origin/blob/master/examples/quickstarts/dancer-mysql.json#L22-L33

Reply:
an aberration on my part, I think; removing it is fine by me.